src/tools/util/MapCanvas.ts

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios'
+import { CustomStageConfig } from '@/util/PointerEventMapper'
+import Konva from 'konva'
+import store from '@/main'
+import { StageGetters } from '@/store/modules/stage'
+import { LayerGetters } from '@/store/modules/layer'
+import { CanvasElementType } from '@/types/Canvas'
+
+export default class MapCanvas {
+  async setMap () {
```

Comment:
Sets a new map to the canvas. This can also be used without having to create a new tactic. It works on its own.